ui/app/components/app/transaction-list-item/index.scss

```
@@ -1,149 +1,49 @@
-      padding: .5rem 1rem;
-      grid-template-columns: 45px 5fr 3fr;
-      grid-template-areas:
-        "nonce nonce nonce nonce"
-        "identicon action estimated-time primary-amount"
-        "identicon status estimated-time secondary-amount";
-      grid-template-rows: auto 24px;
-    }
-
-    &:hover {
-      background: rgba($alto, .2);
-    }
+  &:hover {
+    background-color: $Grey-000;
   }
 
-  &__identicon {
-    grid-area: identicon;
-    grid-row: 1 / span 2;
-    align-self: center;
-
-    @media screen and (max-width: $break-small) {
-      grid-row: 2 / span 2;
-    }
+  &__primaryCurrency {
+    color: $Black-100;
   }
 
-  &__action {
-    text-transform: capitalize;
-    padding: 0 0 4px 0;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    grid-area: action;
-    color: $Grey-800;
-    line-height: 20px;
+  &__secondaryCurrency {
+    color: $Grey-500;
   }
 
-  &__status {
-    grid-area: status;
-    grid-row: 1 / span 2;
-    align-self: center;
-
-    @media screen and (max-width: $break-small) {
-      grid-row: 3;
+  &--pending {
+    color: $Grey-500;
+    &__primaryCurrency {
+      color: $Grey-500;
     }
   }
 
-  &__estimated-time {
-    grid-area: estimated-time;
-    grid-row: 1 / span 2;
-    align-self: center;
-
-    @media screen and (max-width: $break-small) {
-      grid-row: 3;
-      grid-column: 4;
-      font-size: small;
+  &__status {
+    &--unapproved {
+      color: $flamingo;
     }
-  }
-
-  &__nonce {
-    font-size: .75rem;
-    color: #5e6064;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    grid-area: nonce;
-    align-self: start;
-
-    @media screen and (max-width: $break-small) {
-      padding-bottom: 8px;
-      line-height: 12px;
+    &--failed {
+      color: $valencia;
     }
-  }
-
-  &__amount {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    min-width: 0;
-    max-width: 100%;
-
-    &--primary {
-      text-align: end;
-      grid-area: primary-amount;
-      align-self: end;
-      justify-self: end;
-      line-height: 20px;
-
-      @media screen and (max-width: $break-small) {
-        padding-bottom: 4px;
-        height: 100%;
-        color: $Grey-800;
-      }
+    &--cancelled {
+      color: $valencia;
     }
-
-    &--secondary {
-      text-align: end;
-      font-size: .75rem;
-      grid-area: secondary-amount;
-      align-self: start;
-      justify-self: end;
+    &--queued {
       color: $Grey-500;
     }
   }
 
-  &__retry {
-    background: #d1edff;
-    border-radius: 12px;
-    font-size: .75rem;
-    padding: 4px 12px;
-    cursor: pointer;
-    margin-top: 8px;
-
-    @media screen and (max-width: $break-small) {
-      font-size: .5rem;
-    }
-  }
-
-  &__expander {
-    max-height: 0px;
-    width: 100%;
-    overflow: hidden;
-
-    &--show {
-      max-height: 1000px;
-      transition: max-height 700ms ease-out;
+  &__pendingActions {
```

Can we use kebab case for these selectors?

e.g.:

```suggestion
  &__pending-actions {
```

---

Similarly for:

- `__primaryCurrency`
- `__secondaryCurrency`

---

i also fixed a camelCase classname on the list component from the previous pr, but its basically unrelated to this work.